### PR TITLE
chore: bump to nightly-2023-09-13

### PR DESCRIPTION
**Pull request by kim-em**
https://github.com/leanprover/lean4/pull/2480 has landed in `master`, this PR is just waiting until that arrives in a release candidate that we will bump to.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /lake-packages
 /.cache
 .DS_Store
+/lakefile.olean
```
Comment (jcommelin) [inline .gitignore:5]: Should we just ignore `*.olean` in general?

**Review by jcommelin — APPROVED** [review]
bors d+

File: .gitignore (hunk 1 above)
Comment (semorrison) [reply .gitignore:5]: Other oleans are, for now, a sign of something being wrong, so I'd prefer not to ignore them.

**Comment by semorrison** [comment]
Closed in favour of #7174.

**Pull request closed without merging**